monitoring/definitions/shared/codeintel.go

```diff
@@ -587,6 +587,38 @@ func (codeIntelligence) NewGitserverClientGroup(containerName string) monitoring
 	})
 }
 
+// src_codeintel_repoupdater_total
+// src_codeintel_repoupdater_duration_seconds_bucket
+// src_codeintel_repoupdater_errors_total
+func (codeIntelligence) NewRepoUpdaterClientGroup(containerName string) monitoring.Group {
+	return Observation.NewGroup(containerName, monitoring.ObservableOwnerCodeIntel, ObservationGroupOptions{
+		GroupConstructorOptions: GroupConstructorOptions{
+			Namespace:       "codeintel",
+			DescriptionRoot: "repoUpdater client",
+			Hidden:          true,
+
+			ObservableConstructorOptions: ObservableConstructorOptions{
+				MetricNameRoot:        "codeintel_repoupdater",
+				MetricDescriptionRoot: "client",
```

Comment:
```suggestion
				MetricDescriptionRoot: "Repo-updater client",
```

I think this is the right place to add this? Right now the generated description reads:

> Aggregate client operation errors every 5m

Which is a big vague

Reply:
This would be consistent with what already exists. Heres a screenie
![image](https://user-images.githubusercontent.com/18282288/134042895-c7fe980d-ddde-4c1a-880f-233f8a2b6cbd.png)

Reply:
Ah - I might advocate for it to all be changed to be slightly more specific? 🙃 and/or expanded upon like in https://github.com/sourcegraph/sourcegraph/pull/25120/files#r712350821